website/src/content/docs/guides/addons/addon-configuration.md

```diff
@@ -0,0 +1,46 @@
+---
+title: General Addon Configuration
```

Comment:
@alfi0812 

This is not "general configuration" but "advanced configuration". Users should only be touching this if they know what they are doing.